internal/manifest/log.go

```diff
@@ -11,8 +13,5 @@ type keyer interface {
 }
 
 func (l Log) For(kind string, keyer keyer) Log {
-	if kind != "" {
-		kind += " "
-	}
-	return Log(func(f string) { l.Printf("%s%v: %v", kind, keyer.GetKey(), f) })
+	return Log(func(f string) { l.Printf("%s %q: %v", kind, keyer.GetKey(), f) })
```

Comment:
if `kind` is empty you'll have an extra space in the beginning

Reply:
Yes, but in AK `kind` should never be empty.